controller/tests/array_action/svc/array_mediator_svc_test.py

```diff
@@ -587,22 +597,34 @@ def test_validate_supported_space_efficiency_success(self):
         self.svc.validate_supported_space_efficiency(compressed_space_efficiency)
         deduplicated_space_efficiency = config.SPACE_EFFICIENCY_DEDUPLICATED
         self.svc.validate_supported_space_efficiency(deduplicated_space_efficiency)
+        deduplicated_thin_space_efficiency = config.SPACE_EFFICIENCY_DEDUPLICATED_THIN
+        self.svc.validate_supported_space_efficiency(deduplicated_thin_space_efficiency)
+        deduplicated_compressed_space_efficiency = config.SPACE_EFFICIENCY_DEDUPLICATED_COMPRESSED
+        self.svc.validate_supported_space_efficiency(deduplicated_compressed_space_efficiency)
+
+    def _test_test_build_kwargs_from_parameters(self, space_efficiency, pool, name, size, thin=False, compressed=False,
+                                                deduplicated=False):
+        expected_kwargs = {'name': name, 'unit': 'b', 'size': size, 'pool': pool}
+        if thin:
+            expected_kwargs.update({'thin': True})
+        if compressed:
+            expected_kwargs.update({'compressed': True})
+        if deduplicated:
+            expected_kwargs.update({'deduplicated': True})
```

Comment:
can't we just have a `space_efficiency_kwargs` dict parameter, to avoid this code?

Reply:
there are 5 options (thin, compressed, deduplicated, dedup_thin, dedup_compressed), its not really better IMO..

Reply:
I just don't think there is a need for this code

Reply:
done